test/controllers/organizations/donate_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+
+class Organization::DonateControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    Current.organization = create(:organization)
+
+    @admin_person = create(:person)
+    @admin_person.groups << create(:group, :admin, organization: Current.organization)
```

Comment:
We have traits for each group(adopter, admin...) via Factory Bot. Let just use `@admin` to keep this consistent with our other tests. Organization is handled by acts_as_tenant.

```suggestion
    @admin = create(:person, :admin)
```